src/test/java/pageObjectTests/MainPageTest.java

```diff
@@ -0,0 +1,104 @@
+package pageObjectTests;
+
+import io.qameta.allure.*;
+import org.openqa.selenium.*;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.testng.annotations.*;
+import pageObjectClasses.*;
+import utils.DriverSetUp;
+
+import static org.hamcrest.MatcherAssert.*;
+import static org.hamcrest.Matchers.*;
+
+public class MainPageTest extends DriverSetUp {
+    private static final Logger LOGGER
+            = LoggerFactory.getLogger(MainPageTest.class);
+    private CreatedUserAccount createdUser;
+    private String createdUserEmail;
+    private String createdUserPassword;
+    private MainPage mainPage;
+
+    @Attachment(value = "Description {txt}", type = "text/plain", fileExtension = ".txt")
+    public String attachTxt(String txt) {
+        return txt;
+    }
+
+    @Attachment(value = "Screenshot", type = "image/png", fileExtension = ".png")
+    public byte[] takeWholePageScreenshotPNG(WebDriver driver) {
+        return ((TakesScreenshot) driver).getScreenshotAs(OutputType.BYTES);
+    }
+
+    @Step("Test user is created")
+    public void createTestUser() {
+        mainPage = loadMainPage();
+        createdUser = mainPage.clickSignUpLink().continueRegistration("12345")
+                .register("John", "Smith", "john.smith@test.com", "12345", "12345");
+        createdUserEmail = createdUser.getUserAccountEmail();
+        createdUserPassword = createdUser.getUserAccountPassword();
+        attachTxt("Created user email: " + createdUserEmail + " \n" + "Created user password: " + createdUserPassword);
+        takeWholePageScreenshotPNG(driver);
+        LOGGER.info("User is created with email = {}, password = {}", createdUserEmail, createdUserPassword);
+        driver.quit();
+    }
+
+    @Test(description = "Test of Email text on Main page")
+    @Description("Test description: check that email text is correct")
+    public void checkEmailTextIsCorrectTest() {
+        mainPage = loadMainPage();
+        String emailText = mainPage.getEmailFieldText();
+        assertThat(emailText, equalTo("Email"));
+    }
+
+    @Test(description = "Test of Password text on Main page")
+    @Description("Test description: check that password text is correct")
+    public void checkPasswordTextIsCorrectTest() {
+        mainPage = loadMainPage();
+        String passwordText = mainPage.getPasswordFieldText();
+        assertThat(passwordText, equalTo("Password"));
+    }
```

Comment:
>  mainPage = loadMainPage();
>  String passwordText = mainPage.getPasswordFieldText();
>  assertThat(passwordText, equalTo("Password"));

the same here and then below - could be simplified and be more clear

Reply:
Changed to requested in 650df7fc4fecbff7c663b7d51b9574ad6cdd033f.